### ISSUE TITLE
Support for Windows in educational environment

### ISSUE THREAD

**Issue by aitormf** [JdeRobot/JdeRobot#544]
To add support for Windows in educational environment, we decided use only Python libs because it is better that C++ to multiplatform. To complete this we need add support for Windows in Python libs and use [Docker](http://www.docker.com/) (Type of Virtual Machine) to run gazebo. In addition we will use [gzweb ](http://gazebosim.org/gzweb) (web client of gazebo) to not need to use graphical environment in the virtual machine.
I first put the task list and then a explanation of how the process will

**Task list:**
- [x] Python Libs
- [x] Docker
- [x] Docker + GzWeb

## Python Libs
Only I need edit easyiceconfig lib because it has harcoded paths that no exists in Windows

## Docker
I have a [first version](https://hub.docker.com/r/aitormf/jderobot/tags/) (tag base) with gazebo7 and Ice 3.6 but without gzweb. Gazebo7 for testing with views to the jump and Ice 3.6 because in Windows is very difficult install ice 3.5 and easy 3.6. if we go to 3.6, it was not worth spending time fighting to install 3.5.

### GzWeb
Now I have some problems:

- When I try to prepare gzweb I get the following error:
```
/usr/bin/env: node: No existe el archivo o el directorio
/usr/bin/env: node: No existe el archivo o el directorio
There are node-gyp build errors, exiting.
```
this is solved with:

`ln -s /usr/bin/nodejs /usr/bin/node`

- Gzweb didn't show drone models: This is because green and red missing a closing tag and because in the model of three drones had spaces between the label and its content and the customer parsed wrong. for example i had chaged this:
```
<geometry>
    <box>
        <size> 0.001000 0.001000 0.001000 </size>
    </box>
</geometry>
```

by this:
```
<geometry>
    <box>
        <size>0.001000 0.001000 0.001000</size>
    </box>
</geometry>
```

- Gzweb not shows well some worlds: I have put a [question ](http://answers.gazebosim.org/question/14373/gzweb-not-load-well-my-world/)in the platform of gazebo to see if me help to solve it

**Comment by aitormf** [comment]
I've created a pull request to fix a bug that gzweb had with another already had done to solve that the roads would not be, gzweb shows all our worlds

**Comment by aitormf** [comment]
this issue is resolved with #546 #543 and [73 of TeachingRobotics ](https://github.com/JdeRobot/TeachingRobotics/pull/73)
